backend/src/main/java/ToDoList/backend/model/Board.java

```diff
@@ -0,0 +1,120 @@
+package TodoList.backend.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Transient;
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+// This class describes:
+// - The entities of the table Board
+// - The getters and setters
+// - The methods to construct a row in the table Board
+@Entity
+public class Board {
+
+//    This will create a sequence table named lst_sequence that keep track of the next id value to be inserted.
+//    I do not see any benefit using this at the moment.
+//    @Id
+//    @SequenceGenerator(
+//            name = "lst_sequence",
+//            sequenceName = "lst_sequence",
+//            allocationSize = 1
+//    )
+//    @GeneratedValue(
+//            strategy = GenerationType.SEQUENCE,
+//            generator = "lst_sequence"
+//    )
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "board_id")
+    private Long board_id;
+    private String title;
+    private String information;
+    private LocalDateTime created_at;
+    private LocalDateTime last_edited_at;
+    @Transient
+    private Long live_time_day;
+
+    public Board(String title, String information, LocalDateTime created_time, LocalDateTime last_edited_time) {
+        this.title = title;
+        this.information = information;
+        this.created_at = created_time;
+        this.last_edited_at = last_edited_time;

```

Comment:
Java use camalCase. I don't mind you use snake cases in MySQL as field name, but please follow the naming convention in Java.

Reply:
Using lombok would be an alternative.

Reply:
I followed a tutorial to understand the structure of the entity class, my next step is to find a tutorial in Lombok and phase out these boiler plate codes.